batchupload/make_info.py

```diff
@@ -245,27 +247,36 @@ def handle_args(args):
                 options['in_file'] = common.convert_from_commandline(value)
             elif option == '-base_name':
                 options['base_name'] = common.convert_from_commandline(value)
+            elif option == '-update_mappings':
+                if value.lower() in ('f', 'false', 'no', 'n'):
```

Comment:
So if the user uses the argument with a value that is not specified here, it sets "update_mappings" to true. Might be tricky if the intent was to disable it but there was a typo. Maybe there should be a closed set of positive arguments as well, that way both options are treated the same way. Also saves the effort of having to remember/look up in usage which option is default.